src/java/org/apache/sqoop/util/SqoopJsonUtil.java

```diff
@@ -40,6 +40,9 @@ private SqoopJsonUtil() {
   }
 
   public static String getJsonStringforMap(Map<String, String> map) {
+    if (map == null) {
```

Comment:
Nit: it is a good practice not to reassign the input parameters so I would change this to something like the following:
`Map<String, String> mapToUse = (map == null) ? emptyMap() : map;
  JSONObject pathPartMap = new JSONObject(mapToUse);`

Reply:
Thanks for your review and suggestion, I have updated the commit and [SQOOP-3435](https://issues.apache.org/jira/browse/SQOOP-3435)